nsv13/code/modules/munitions/ammunition/missiles/missile_types.dm

```diff
@@ -40,6 +40,7 @@
 /obj/item/projectile/guided_munition/missile/dud
 	icon_state = "torpedo_dud"
 	damage = 0
+	can_home = FALSE
```

Comment:
I think it would be funny if this updated when you've installed the guidance computer and propulsion system.

Reply:
Done.